databank/management/commands/sources/FDRS.py

```diff
@@ -120,11 +121,15 @@ def load(country, overview, fdrs_data):
     if country.iso is None or fdrs_data is None:
         return
 
+    get_max_year = lambda data: max(int(item['year']) for item in fdrs_data.values())
+    fdrs_data_fetched_year = get_max_year(fdrs_data)
```

Comment:
Any reason why we need lambda here instead of simple statement?

Reply:
Just creating this to look clean.

Reply:
Replaced with normal statement